### PR TITLE
Add in support for a google maps api key.

### PR DESCRIPTION
**Pull request by davidshorter**
Pass in google maps api key for the mapping component on the contact us page.
Please advise if there is a better way to do this.

### CODE REVIEW

**Comment by adrianmo** [comment]
@davidshorter Thanks for the contribution. It's a nice feature. 

Just a minor thing, can you please rename the option to `GoogleMapsApiKey`? I think it is a more descriptive name. 

Thanks,
-Adrian

File: exampleSite/config.toml [hunk 1]
```diff
@@ -14,6 +14,9 @@ paginate = 10
     viewMorePostLink = "/blog/"
     author = "DevCows"
 
+	# Google Maps license key (if not set will default to not passing a key.)
+    googleMapsLicense = ""
```
Comment (adrianmo) [inline exampleSite/config.toml]: Rename to `googleMapsApiKey`

**Comment by computerfr33k** [comment]
Not sure if it Is it just me, but when I enter in my google maps api key in the config file, hugo never detects that param as being set. I am using hugo v0.18.1

I have tried outputting the contents of the param variable using `{{ echoParam .Site.Params "googleMapsApiKey" }}` but nothing is printed out.

But if I change the variable name to `google_maps_api_key` then it will output the contents.

**Comment by davidshorter** [comment]
@computerfr33k 

Have fixed the above error as part of PR https://github.com/devcows/hugo-universal-theme/pull/60

**Pull request merged**